Translation: Update some variable names

Dirain1700 opened the pull request:
Deleted these lines:
```diff
- "User ${name} must be in the battle room already.": "",
- "${name} was added to the battle as Player ${playerNum} by ${user.name}.": "",
```

fixed this line:
```diff
- "${targetUser.name} was kicked from a battle by ${user.name} ${displayTarget}": "",
+ "${targetUser.name} was kicked from a battle by ${user.name} ${displayReason}": "",
```

ref: #8303 

I couldn't catch all changes; please tell me :)

AnnikaCodes commented on translations/english/core-commands.ts:
```diff
@@ -88,14 +88,13 @@ export const translations: Translations = {
 		"You can only do this in unrated non-tour battles.": "",
 		"User ${targetUsername} not found.": "",
-		"${targetUser.name} was kicked from a battle by ${user.name} ${displayTarget}": "",
+		"${targetUser.name} was kicked from a battle by ${user.name} ${displayReason}": "",
```
It doesn't matter what the variables are called in translation. `displayReason` is clearer, though, so I'm fine with it.

Dirain1700 replied:
Oh, Thank you!
(I'm just modifying it to reflect the changes)

Dirain1700 replied:
I'm just modifying it to reflect the changes :)

AnnikaCodes replied:
It... doesn't need to reflect the changes, but this is fine either way.

Slayer95 commented:
> I couldn't catch all changes; please tell me :)

Is anybody willing to collaborate? If not, this surely can be merged as-is as a follow-up to the linked refactor PR.

Dirain1700 commented:
Please use this in commit message:

```txt
Deleted these lines:
"Player must be set to \"p1\" or \"p2\", not \"${target}\".": "",
"User ${name} must be in the battle room already.": "",
"This room already has a player in slot ${target}.": "",
"${targetUser.name} is already a player in this battle.": "",
"${name} was added to the battle as Player ${playerNum} by ${user.name}.": "",

fixed this line:
- "${targetUser.name} was kicked from a battle by ${user.name} ${displayTarget}": "",
+ "${targetUser.name} was kicked from a battle by ${user.name} ${displayReason}": "",

ref: #8303
```

AnnikaCodes commented on translations/english/core-commands.ts:
```diff
@@ -83,19 +83,14 @@ export const translations: Translations = {
 		"${user.name} hid the replay of this battle.": "",
 		"You can only do this in battle rooms.": "",
 		"You can only add a Player to unrated battles.": "",
-		"Player must be set to \"p1\" or \"p2\", not \"${target}\".": "",
```
This translation seems to be still in use: https://github.com/smogon/pokemon-showdown/blob/0475b38ae8ad2cd1ea0b2849972a9c821bd6a1de/server/chat-commands/core.ts#L952

AnnikaCodes commented on translations/english/core-commands.ts:
```diff
@@ -85,17 +85,13 @@ export const translations: Translations = {
 		"Player must be set to \"p1\" or \"p2\", not \"${target}\".": "",
 		"User ${name} not found.": "",
-		"User ${name} must be in the battle room already.": "",
-		"This room already has a player in slot ${target}.": "",
```
This translation is also still in use: https://github.com/smogon/pokemon-showdown/blob/5105cc46a377e3effbb3cf5afb9b0ee826b29593/server/chat-commands/core.ts#L968

AnnikaCodes commented on translations/english/core-commands.ts:
```diff
@@ -85,17 +85,13 @@ export const translations: Translations = {
 		"User ${name} not found.": "",
-		"User ${name} must be in the battle room already.": "",
-		"This room already has a player in slot ${target}.": "",
-		"${targetUser.name} is already a player in this battle.": "",
```
As is this one: https://github.com/smogon/pokemon-showdown/blob/5105cc46a377e3effbb3cf5afb9b0ee826b29593/server/chat-commands/core.ts#L976

AnnikaCodes commented:
Thanks!
